templates/tfengine/recipes/project.hcl

```diff
@@ -84,6 +84,7 @@ schema = {
             properties = {
               api = {
                 type = "string"
+                description = "The API to apply the roles to. Example: healthcare.googleapis.com"
```

Comment:
This should be "The API whose default Service Agent will be force-created and granted the role. Example: healthcare.googleapis.com."

You can update it in a separate PR.